build.sbt

```diff
@@ -40,9 +40,13 @@ inThisBuild(
         "-feature",
         "-unchecked",
         "-Xsource:3",
-        "-Wunused:imports",
         "-Wconf:cat=unused-imports&src=src_managed/.*:silent"
-      )
+      ) ++ (CrossVersion.partialVersion(scalaVersion.value) match {
+        case Some((2, 12)) =>
+          List("-Ywarn-unused:imports")
+        case Some((2, 13)) =>
+          List("-Wunused:imports")
```

Comment:
So I have no idea what is going on here. `-Wunused:imports` should only be added for Scala 2.13 but for some reason its also being added for scala 2.12 (I have used this technique for countless other projects without issues).

Is there something I am missing here?